### PR TITLE
Welcome Modal

### PR DESCRIPTION
**Pull request by ifunanyaScript**

### CODE REVIEW

File: components/welcome/index.tsx [hunk 1]
```diff
@@ -0,0 +1,66 @@
+'use client';
+import { useState } from 'react';
+import { AnimatePresence, motion } from 'framer-motion';
+
+import { TextSampleIcon } from '@/public/icons';
+import SMClickAnimation from '../click-animation';
+
+const texts = [
+  'Once your PR is merged, the token list will update automatically to include your token and your token will be available on the Base Bridge.',
```
Comment (njokuScript) [inline components/welcome/index.tsx]: **Change to:**

Once your PR is merged, the token list will update automatically to include your token on our bridging interface.

File: components/welcome/index.tsx [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+  'To complete the migration, submit a pull request to the Superchain token repo list.',
+  'For support please reach out to us via our telegram',
```
Comment (njokuScript) [inline components/welcome/index.tsx]: For support please reach out to us via our telegram channel: https://t.me/+8vDPDkrN_-gwZTA8

File: components/welcome/index.tsx [hunk 3]
```diff
@@ -0,0 +1,66 @@
+'use client';
+import { useState } from 'react';
+import { AnimatePresence, motion } from 'framer-motion';
+
+import { TextSampleIcon } from '@/public/icons';
+import SMClickAnimation from '../click-animation';
+
+const texts = [
+  'Once your PR is merged, the token list will update automatically to include your token and your token will be available on the Base Bridge.',
+  'To complete the migration, submit a pull request to the Superchain token repo list.',
```
Comment (njokuScript) [inline components/welcome/index.tsx]: To complete the migration, make sure you verify the migration by adding a link to the verification tweet on the pull request

**Review by njokuScript — CHANGES_REQUESTED** [review]
Check the comments I left on the welcome index file @ifunanyaScript

File: components/welcome/index.tsx (hunk 1 above)
Comment (ifunanyaScript) [reply components/welcome/index.tsx]: Done

File: components/welcome/index.tsx (hunk 2 above)
Comment (ifunanyaScript) [reply components/welcome/index.tsx]: Done

File: components/welcome/index.tsx (hunk 3 above)
Comment (ifunanyaScript) [reply components/welcome/index.tsx]: Done

**Review by MeisterJustice — APPROVED** [review]

**Pull request merged**